.github/workflows/check-catalyst.yaml

```
@@ -528,30 +566,33 @@ jobs:
       - name: Build Runtime test suite for Lightning simulator
         if: ${{ matrix.backend == 'lightning' }}
         run: |
-            C_COMPILER=$(which gcc) \
-            CXX_COMPILER=$(which g++) \
-            COMPILER_LAUNCHER="" \
-            make coverage-runtime
-            mv runtime/build/coverage.info coverage-${{ github.job }}.info
+          C_COMPILER=$(which ${{ needs.constants.outputs[format('c_compiler.{0}', matrix.compiler)] }}) \
+          CXX_COMPILER=$(which ${{ needs.constants.outputs[format('cxx_compiler.{0}', matrix.compiler)] }}) \
```

I may revert this to gcc/g++ because `lcov` (the lib used to create C++ code coverage via `make coverage-runtime`) only works with gcc/g++ (as of this dependency to `gcov`).   
```suggestion
          C_COMPILER=$(which gcc) \
          CXX_COMPILER=$(which g++) \
```

---

Good point 👍